packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -12,6 +12,11 @@ const defaultConfigs = {
       { type: "cryptowatch", exchange: "bitstamp" }
     ]
   },
+  "XIO/ETH": {
```

Comment:
Hey @anassohail99! Typically this file is reserved for mainnet price feed configs. It shouldn't be necessary to merge the Kovan price feed into protocol - but definitely a good thing to test with on your side!

Happy to review further if you want to update this to the mainnet XIOETH pool [here](https://etherscan.io/address/0xe0cc5afc0ff2c76183416fb8d1a29f6799fb2cdf).